src/app/auth/signup/signup.component.ts

```diff
@@ -1,56 +1,79 @@
-import { Component } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
 import {
   FormBuilder,
-  FormGroup,
   ReactiveFormsModule,
   Validators,
 } from '@angular/forms';
-import { CommonModule } from '@angular/common';
-import { Router, RouterLink } from '@angular/router';
-import { AppState, User } from '../../app.state';
-import { select, Store } from '@ngrx/store';
-import { Observable, tap } from 'rxjs';
-import { isLoggedIn } from '../auth.selectors';
-import { AuthService } from '../../guard/auth.service';
+import {CommonModule} from '@angular/common';
+import {Router, RouterLink} from '@angular/router';
+import {AppState} from '../../app.state';
+import {select, Store} from '@ngrx/store';
+import {Observable, tap} from 'rxjs';
+import {isLoggedIn} from '../auth.selectors';
+import {AuthService} from '../../guard/auth.service';
+import {CustomInputFieldComponent} from '../custom-input-field/custom-input-field.component';
 
 @Component({
   selector: 'app-signup',
   standalone: true,
-  imports: [ReactiveFormsModule, RouterLink, CommonModule],
+  imports: [ReactiveFormsModule, RouterLink, CommonModule, CustomInputFieldComponent],
```

Comment:
Do you still need CommonModule here

Reply:
Yes I am using an ngClassin the html template

Reply:
Can we use the class binding syntax instead